filesystem/gpt_partition_table.ksy

```diff
@@ -1,17 +1,21 @@
 meta:
   id: gpt_partition_table
   endian: le
-  # spec from https://en.wikipedia.org/wiki/GUID_Partition_Table
-  # For now we only allow for 512 byte sectors.
-  # For 4096 byte sectors, change 0x200 to 0x1000 in instances.primary.pos, instances.backup.pos and types.partition_header.instances.entries.pos 
+  # License: Copyright and related rights waived via CC0 [ https://creativecommons.org/publicdomain/zero/1.0/ ]
+  # Specification taken from https://en.wikipedia.org/wiki/GUID_Partition_Table
+  #
+  # The default sector_size is 512 bytes to parse 4096 byte sectors
+  # Set both instances.sector_size.value and types.partition_header.instances.sector_size.value to 0x1000
```

Comment:
Ahem. You don't really have to resort to YAML comments to add licensing info and documentation. There's literally `license` and `doc` fields to do that. Check out how it's done in [mbr_partition_table](https://github.com/kaitai-io/kaitai_struct_formats/blob/master/filesystem/mbr_partition_table.ksy#L1-L17), for instance.